pages/models/regulations/governors/GovSteamEU/index.md

```diff
@@ -0,0 +1,422 @@
+---
+layout: page
+title: GovSteamEU
+tags: [regulations, governor, controller, synchronous machine, Opensource, IEC, CIMDynamic, RMS, modelica, dynawo]
+author: Martin Franke, Saikrishna Vallabhaneni
+date: 20/06/2024
+last-updated: 20/06/2024
+id: #59
+---
+# GovSteamEU
+
+## Context
+
+The traditional operation of electrical power systems depends on the
+performance of the synchronous generators of thermal plants, typically
+driven by various types of turbines such as steam, gas, or hydro
+turbines. Steam turbines are commonly found in coal, oil, and nuclear
+power plants. The process begins with the generation of high-pressure
+high-temperature steam in a boiler. A governor regulates the mechanical
+power ($$P_m$$) delivered from the turbine to the electrical generator.
+GovSteamEU is a generic steam turbine model, can be found in [1], and
+includes the mechanical part of the turbine with high and low pressure
+stages.
+
+## Model use, assumptions, validity domain and limitations
+
+The model is a positive-sequence RMS model, hence it assumes symmetrical
+operating conditions and neglects high-frequency dynamics. This type of
+model is often used in large-scale stability studies, for which it
+reflects the relevant phenomena. It is not a detailed physical model of
+the unit. Also for some stability phenomena (e.g. resonance stability)
+this model is not sufficient and EMT models or other approaches may be
+necessary .
+
+The model operates only under the assumption that resistive and
+mechanical losses are negligible, and in steady state condition
+$$P_\mathrm{e} = P_\mathrm{m}$$. This assumes that mechanical losses do
+not have any major effect on dynamic stability [1].
+
+## Model description
```

Comment:
could we have a sheme of the overall model before the different descriptions?

Reply:
I moved the Model Schema up.